spec/requests/certificates/secondary_certificate/show_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 RSpec.describe Certificates::SecondaryCertificateController do
   let(:user) { create(:user) }
+  let!(:cs_accelerator) { create(:cs_accelerator) }
```

Comment:
Secondary has a programme_objective which requires CSAccelerator programme to be present. Ideally we should have a better way of seeding the programmes like they are in production in our tests. Without putting together that system, I think its adequate to just create cs_accelerator manually a head of time.